HLTrigger/Configuration/python/HLT_75e33/modules/hltParticleFlowSuperClusterHGCalFromTICLUnseeded_cfi.py

```diff
@@ -1,15 +1,15 @@
 import FWCore.ParameterSet.Config as cms
 
-particleFlowSuperClusterHGCalFromTICLUnseeded = cms.EDProducer("PFECALSuperClusterProducer",
+hltParticleFlowSuperClusterHGCalFromTICLUnseeded = cms.EDProducer("PFECALSuperClusterProducer",
     BeamSpot = cms.InputTag("hltOnlineBeamSpot"),
     ClusteringType = cms.string('Mustache'),
     ESAssociation = cms.InputTag("hltParticleFlowClusterECALUnseeded"),
     EnergyWeight = cms.string('Raw'),
     PFBasicClusterCollectionBarrel = cms.string('particleFlowBasicClusterECALBarrel'),
     PFBasicClusterCollectionEndcap = cms.string(''),
     PFBasicClusterCollectionPreshower = cms.string('particleFlowBasicClusterECALPreshower'),
-    PFClusters = cms.InputTag("particleFlowClusterHGCalFromTICLUnseeded"),
-    PFSuperClusterCollectionBarrel = cms.string('particleFlowSuperClusterECALBarrel'),
+    PFClusters = cms.InputTag("hltParticleFlowClusterHGCalFromTICLUnseeded"),
+    PFSuperClusterCollectionBarrel = cms.string('hltParticleFlowSuperClusterECALBarrel'),
```

Comment:
As far as I can tell, I agree with @waredjeb regarding consistency.
On the other hand, this product (I mean the barrel one created by this HGCal-labelled module) is never consumed. So any name would do w/o crashes.

Reply:
Thank @rovere for the check.

>  On the other hand, this product (I mean the barrel one created by this HGCal-labelled module) is never consumed. So any name would do w/o crashes.

I would propose to go ahead with the PR as is now to avoid re-triggering a lot of signatures and then fix this immediately after, if people agree.

Reply:
Fixed at https://github.com/cms-sw/cmssw/pull/45768. Let me know if you spot other problems.